engine/src/main/java/org/hibernate/search/query/dsl/PlainTextQueryDefinitionTermination.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.search.query.dsl;
+
+
+public interface PlainTextQueryDefinitionTermination {
+
+	/**
+	 * Value searched in the fields.
+	 * The value is passed to the respective fields analyzers.
```

Comment:
This javadoc should be rephrased: the value is parsed, it's not analyzed by the field analyzers. Only part of it is actually analyzed.